### PR TITLE
Fix issue 19492 - Catalog Product - Customizable Options label alignment issue 

### PR DESCRIPTION
**Pull request by SyntaxKay**
### Description (*)

Magento 2.3-develop 
 Catalog Product - Customizable Options label alignment issue 
### Fixed Issues (if relevant)

1.magento/magento2#19492 : Catalog Product - Customizable Options label alignment issue 

### Manual testing scenarios (*)

1.N/A

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @speedy008. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by orlangur** [comment]
@speedy008 could you please make one commit instead of four?

File: app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less [hunk 1]
```diff
@@ -671,9 +671,12 @@
                 margin: 0;
                 opacity: 1;
                 position: static;
-                text-align: left;
+
```
Comment (sivaschenko) [inline app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less]: Please remove empty line

**Review by sivaschenko — CHANGES_REQUESTED** [review]
@speedy008 thanks for the contribution!  Please remove the blank line and squash changes to a single commit.

File: app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less (hunk 1 above)
Comment (sivaschenko) [inline app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less]: Please remove the blank line

**Comment by SyntaxKay** [comment]
Hi @sivaschenko 
I have updated code kindly review it.

**Comment by sivaschenko** [comment]
@speedy008 thanks for the update, can you also take a look at https://github.com/magento/magento2/pull/19493#issuecomment-443404710 comment. Please squash commits into a single one

**Comment by SyntaxKay** [comment]
Hi @VladimirZaets 
I have Squash 4 commit to one.

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-3659](https://jira.corp.magento.com/browse/ENGCOM-3659) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @speedy008. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**